### PR TITLE
Upgrade Ruby to `2.7.4` and gem dependencies (OWO)

### PR DESCRIPTION
**Pull request by waldofouche**
Updates Ruby, Rails and gem dependencies:

- Ruby from 2.7.2 to 2.7.4
- All other gem dependencies to latest versions

`ExecJS` was kept at it's current version. Will be updated when moved to NWO

This is part of https://github.com/ausaccessfed/stockpile/issues/87

### CODE REVIEW

File: config/environments/development.rb [hunk 1]
```diff
@@ -32,6 +32,7 @@
 
   config.shib_rack.development_mode = true
 
+  # rubocop:disable Naming/InclusiveLanguage
```
Comment (ritchiey) [inline config/environments/development.rb:35]: Does anyone mind if we just disable `Naming/InclusiveLanguage` globally? It doesn't seem to have anything to do with software quality.

**Review by ritchiey — APPROVED** [review]
Nice one!

File: config/environments/development.rb (hunk 1 above)
Comment (waldofouche) [reply config/environments/development.rb:35]: I agree with you on this one, what are your thoughts @rianniello / @ArthurZheng ?

Comment (ArthurZheng) [reply config/environments/development.rb:35]: agree 👍

**Review by ritchiey — APPROVED** [review]

**Pull request merged**